crowbar_framework/app/controllers/api/ceph_controller.rb

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2016, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Api
+  class CephController < ApiController
+    before_action :set_ceph
+
+    api :GET, "/api/ceph/repocheck", "Sanity check ceph repositories"
+    api_version "2.0"
+    def repocheck
+      render json: @ceph.repocheck
+    end
+
+    protected
+
+    def set_ceph
+      @ceph = Api::Ceph.new
```

Comment:
I know this PR isn't introducing or even really changing this, but I don't understand why it's an instance method or why an instance variable is being assigned, since I can't see any instance-specific state.  Am I missing something?